script.js

```
@@ -0,0 +1,159 @@
+            case 'cmhc-is':
+                // "Balanced market" - let's assume a more modest 1.8% annual growth.
+                annualGrowthRate = 0.018;
+                break;
+            case 'imf':
+                const popGrowth = parseFloat(popGrowthSlider.value);
+                const interestRate = parseFloat(interestRateSlider.value);
+                const newSupply = parseInt(newHousingSlider.value, 10);
+
+                // Formula: price growth = 1.5% + (1.5 * Pop. Growth) - (2.0 * Interest Rate) - (0.0005 * (New Supply - 4700))
+                console.log(`JS DEBUG: popGrowth=${popGrowth}, interestRate=${interestRate}, newSupply=${newSupply}`);
+                const growthPercent = 1.5 + (1.5 * popGrowth) - (2.0 * interestRate) - (0.0005 * (newSupply - 4700));
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)